Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -7639,3 +7639,100 @@
     engine: Blink
     engine_version: 84.0.4147.38
     family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10; SM-J600G Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/81.0.4044.138 Mobile Safari/537.36
+  client:
+    type: browser
+    name: World Browser - Fast & Simple
```

Comment:
Would not "World Browser" suffice? "Fast & Simple" feels unnecessary.

Reply:
Renamed, thank you.